suite-common/thp/src/startThpAutoconnectThunk.ts

```diff
@@ -6,24 +6,29 @@ import TrezorConnect from '@trezor/connect';
 import { finishThpAutoconnectThunk } from './finishThpAutoconnectThunk';
 import { THP_PREFIX, thpActions } from './thpActions';
 
-export const startThpAutoconnectThunk = createThunk<void, void, void>(
+export const startThpAutoconnectThunk = createThunk(
     `${THP_PREFIX}/startThpAutoconnectThunk`,
-    async (_, { getState, dispatch }) => {
+    async (_, { getState, dispatch, rejectWithValue }) => {
         const device = selectSelectedDevice(getState());
 
         if (device === undefined) {
-            return;
+            return rejectWithValue('Device was not found.');
         }
 
         const response = await TrezorConnect.thpGetCredentials({ device });
 
         if (response.success) {
             dispatch(thpActions.addCredential({ credential: response.payload }));
+            dispatch(finishThpAutoconnectThunk());
+
+            return;
         } else {
             dispatch(
                 notificationsActions.addToast({ type: 'error', error: response.payload.error }),
             );
+            dispatch(finishThpAutoconnectThunk());
+
+            return rejectWithValue(response.payload.error);
         }
-        dispatch(finishThpAutoconnectThunk());
```

Comment:
Based on the usage, this shouldn't affect anything on desktop side as the response is not subscribed anywhere.